_1327/information_pages/views.py

```diff
@@ -9,10 +11,21 @@
 def unlinked_list(request):
 	if InformationDocument.objects.exists():
 		permission_name = InformationDocument.objects.first().edit_permission_name  # the property can only be accessed from an instance of the class
-		information_pages = get_objects_for_user(request.user, permission_name, klass=InformationDocument.objects.filter(menu_items__isnull=True)).order_by('title')
+		menu_pages = get_objects_for_user(request.user, permission_name, klass=InformationDocument.objects.filter(is_menu_page=True))
+
+		menu_page_document_ids = set()
```

Comment:
please add a short comment explaining what is happening here, I mean I get it, but it took some time^^